src/many/src/message.rs

```diff
@@ -181,24 +186,131 @@ impl CoseSign1RequestMessage {
         }
     }
 
+    /// Perform WebAuthn request verification
+    ///
+    /// This is non-standard COSE
+    /// See https://webauthn.guide/#webauthn-api
+    fn _verify_webauthn(
+        &self,
+        unprotected: BTreeMap<Label, Value>,
+        key: CoseKeyIdentity,
+    ) -> Result<(), String> {
+        #[derive(Deserialize)]
+        struct ClientData {
+            challenge: String,
+            #[allow(dead_code)]
```

Comment:
Please remove to make sure we actually use the origin

Reply:
Good catch. We should add a clippy directive to catch those if it exists.